test/lib.api.v1.report-controller.test.js

```diff
@@ -183,18 +200,31 @@ describe('Report controller', function() {
           .expect(200)
           .end(function(err, res) {
             expect(res.body).to.have.property('_incident');
-            expect(res.body._incident).to.equal('54c73024ae04d1f9c3a678d6');
+            expect(res.body._incident).to.equal(String(incidents[0]._id));
 
             request(reportController)
             .get('/api/v1/report/' + reports[1]._id)
             .expect(200)
             .end(function(err, res) {
               expect(res.body).to.have.property('_incident');
-              expect(res.body._incident).to.equal('54c73024ae04d1f9c3a678d6');
+              expect(res.body._incident).to.equal(String(incidents[0]._id));
               done();
             });
           });
         });
     });
+    it('should update the totalReports field in incident', function(done) {
+      request(reportController)
+        .patch('/api/v1/report/_link')
+        .send({ ids: [reports[2].id], incident: incidents[0]._id })
+        .expect(200)
+        .end(function(err) {
+          if (err) return done(err);
+          Incident.findById(incidents[0]._id, function(err, incident) {
+            expect(incident.totalReports).to.equal(3);
+            done();
```

Comment:
This test would be even better if we first check `expect(incident.totalReports).to.equal(2);` or (extra nice) if we then remove the report and check that it goes down to 2.